EmotionalLove/Asuna
# Make gradlew executable to avoid running chmod in CI

## Overview

The gradle wrapper (`gradlew`) did not have executable permissions, and it looks like there was a workaround to run `chmod +x` within the CI pipeline.

This PR makes `gradlew` executable to avoid running `chmod` in CI.

## Testing Instructions

Make sure you can run `$ ./gradlew` on your workstation:

```bash
> rbreslow@maiden Asuna (feature/jrb/fix-gradlew-file-permissions) $ ./gradlew
Downloading https://services.gradle.org/distributions/gradle-2.14-all.zip
...
```

See CI checks below.
